.github/workflows/release.yml

```diff
@@ -6,10 +6,12 @@ on:
       - v*
 
 jobs:
-  repository-dispatch:
-    - name: Send repository dispatch
-      run: |
-        curl -XPOST -u "${{ secrets.USERNAME}}:${{secrets.TOKEN}}" -H "Accept: application/vnd.github.everest-preview+json"  -H "Content-Type: application/json" https://api.github.com/repos/ONSdigital/eq-questionnaire-runner/dispatches --data '{"event_type": "schemas_release"}'
+  name: Repository Dispatch
```

Comment:
I think the name of this step should be something like "Notify Runner".

Reply:
Ok, no problem. Changed👍